content/en/docs/concepts/configuration/secret.md

```diff
@@ -43,21 +53,25 @@ The name of a Secret object must be a valid
 The keys of `data` and `stringData` must consist of alphanumeric characters,
 `-`, `_` or `.`.
 
+
 ### Built-in Secrets
 
 #### Service accounts automatically create and attach Secrets with API credentials
 
-Kubernetes automatically creates secrets which contain credentials for
-accessing the API and automatically modifies your Pods to use this type of
-secret.
+Kubernetes automatically creates Secrets that contain credentials for
+accessing the Kubernetes API. By default, the Kubernetes control plane automatically
+modifies newly-added Pods to use this type of Secret.
```

Comment:
nit: double use of "automatically". Could write:
 By default, the Kubernetes control plane modifies newly-added Pods to use this type of Secret.
Which type of Secret?

Reply:
I think "this type of Secret" means service account Secrets.